app/forms/curation_concerns/forms/work_form.rb

```diff
@@ -0,0 +1,83 @@
+module CurationConcerns
+  module Forms
+    class WorkForm
```

Comment:
@jcoyne Looks like they added two methods, `#ordered_fileset_members` and `#ordered_work_members`.